R/round.r

```diff
@@ -159,10 +161,10 @@ parse_unit_spec <- function(unitspec) {
     mult <- as.numeric(parts[[1]])
     unit <- parts[[2]]
   }
-  
+
```

Comment:
They first showed up when I added a new line to the switch statement in ceiling_date and shifted the contents below it. I've never really thought of these whitespace modifications before, but in RStudio, you can discard the these kinds of chunks so they don't appear in the commit.

Is this PR almost finished? Do the tests and docs need to be modified to reflect this addition?